application/templates/secret.yaml

```diff
@@ -15,9 +15,21 @@ metadata:
   annotations:
 {{ toYaml $.Values.secret.annotations | indent 4 }}
 {{- end }}
+{{- if .data }}
 data:
 {{- range $key, $value := .data }}
   {{ $key }}: {{ $value | b64enc }}
 {{- end }}
+{{- else if .stringData }}
+stringData:
+{{- range $key, $value := .stringData }}
+  {{ $key }}: {{ $value }}
+{{- end }}
```

Comment:
`stringData` isn't mutually exclusive with `data` for Secrets. We can **and should** add it alongside `data` and the API will encode the contents to base 64.

Reply:
Done!